### ISSUE TITLE
Aeris dungeon cancels at the end

### ISSUE THREAD

**Issue by Merijnvanesch** [MelvorIdle/melvoridle.github.io#1506]
I am using the android app. Updated fully. In aeris god dungeon halfway through the last boss it just quits the fight for no reason. This happened twice. Very frustrating since I had to go through the entire dungeon again, only for it to fail again.

**Comment by MelvorIdle** [comment]
Were you dying in the dungeon by chance?

**Comment by Merijnvanesch** [comment]
I'm sorry that a serious question?Op 12 nov. 2021 03:37 schreef Melvor Idle ***@***.***>:
Were you dying in the dungeon by chance?

**Comment by MelvorIdle** [comment]
Yes, it is a serious question. No information was provided to help figure out what happened, and the bug report template was not used, so I need to ask these kinds of questions.

**Comment by Merijnvanesch** [comment]
I did not die, I'm sure of that. I tried the bug report but it didn't work very well. It seems to just crash at the same time every time. De dungeon will be cancelled, I still have all hp left, and never lose any stuff except for the 30 minutes I spent on each try. It looks like it happens when too many attacks happen at the same time.GreetsOp 16 nov. 2021 10:14 schreef Melvor Idle ***@***.***>:
Yes, it is a serious question. No information was provided to help figure out what happened, and the bug report template was not used, so I need to ask these kinds of questions.
